### PR TITLE
Fix some grammatical errors to make the doc better

### PR DESCRIPTION
**Pull request by AdamDang**
Fix some grammatical errors to make the doc better

### CODE REVIEW

**Comment by frapposelli** [comment]
/assign @dougm 

PTAL

File: docs/deploying_cloud_provider_vsphere_with_rbac.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Deploying `cloud-provider-vsphere` with RBAC
 
-This document is designed to quickly get you up and running with the `cloud-provider-vsphere` external Cloud Controller Manager (CCM).
+This document is designed to quickly get you setting up and running with the `cloud-provider-vsphere` external Cloud Controller Manager (CCM).
```
Comment (frapposelli) [inline docs/deploying_cloud_provider_vsphere_with_rbac.md]: disclaimer: I'm a non-native speaker

Is `setting` really needed in this sentence? I've always seen the form

> get you up and running

Comment (AdamDang) [reply docs/deploying_cloud_provider_vsphere_with_rbac.md]: Hi, Frapposelli, I've checked and confirmed that you are right. 
Thanks!

**Pull request merged**